.travis.yml

```diff
@@ -2,8 +2,7 @@ language: python
 python:
 - "2.7"
 # install dependency
-# install: "pip install -r requirements.txt"
-install: "echo 'nothing to install for now.'"
+install: pip install -r requirements.txt
 # run tests
 # script: nosetests
```

Comment:
You wrote test script but not modify .travis.yml, so there is no test be run, please fix it , thanks.